components/magic.js

```diff
@@ -0,0 +1,39 @@
+var fetch = require('node-fetch');
+
+async function cardCmd(message, args)
+{
+    let cardName = "";
+    for(let i = 1; i < args.length; i++) {
+        cardName = cardName + args[i] + " ";
+    }
+    searchCard(cardName)
+    .then((card) => {message.channel.send(card.image_uris.normal);})
```

Comment:
You should add some safety checks here:
if (!card || !card.image_uris || !card.image_uris.normal) send error